packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -168,9 +169,10 @@ module.exports = {
             options: {
               // @remove-on-eject-begin
               babelrc: false,
-              presets: [require.resolve('babel-preset-react-app')],
+			  presets: [require.resolve('babel-preset-react-app')],
```

Comment:
Can you please fix the alignment on these?

Reply:
Spacing should be fixed.